src/components/AlbumCard.vue

```diff
@@ -2,13 +2,19 @@
   <VCard :to="{ name: 'album', params: { id: album.id } }">
     <VImg :aspect-ratio="1" :src="album.image500" v-if="album.image500" />
     <VImg :aspect-ratio="1" :src="album.image" v-else-if="album.image" />
-    <VCardTitle class="pb-0">
+    <VImg
+      :aspect-ratio="1"
+      :src="require('../assets/svg/album.svg')"
```

Comment:
I would prefer not to bring parts of the MDI dependency in-tree. Is there no way to use an icon here or get this SVG from the yarn dependency?

Reply:
The dependency didn't include the SVGs. Since the icons behave as text, using that made it a lot more complex for a lesser result

Reply:
https://www.npmjs.com/package/@mdi/svg?